embedding-sdk/authenticated/jwt/readme.md

```diff
@@ -0,0 +1,78 @@
+# MongoDB Charts Embedding Example - Authenticated Embedded Chart
+
+## Background
+
+📄 _[See the MongoDB Charts Embedding Docs for more details](https://docs.mongodb.com/charts/saas/embedding-charts/)_
+
+🎮 _[Play with a live demo of this sample here](https://codesandbox.io/s/github/mongodb/charts-embedding-examples/tree/feature/CHARTS-3248-embedded-charts-jwt-demo/embedding-sdk/authenticated/jwt)_
```

Comment:
This isn't working for me yet - do I need to change the package version?

Reply:
I'll have to ping Scott about pushing the update to the SDK, but yes it doesn't work for me either since i've tested with the dev package version.

Reply:
Demo is working now :)